src/app/profile/yourGiving/stopStartRecurringGifts/restartGift/step1/suspendedGifts/suspendedGifts.tpl.html

```diff
@@ -0,0 +1,57 @@
+<div class="modal-header">
+  <div class="container-fluid">
+    <div class="row">
+      <div class="col-xs-12">
+        <div class="border-bottom-small">
+          <button type="button" class="close" ng-click="$ctrl.cancel()" aria-label="{{'Close' | translate}}">
+            <span aria-hidden="true">×</span>
+          </button>
+          <h3 translate>Restart A Gift</h3>
+          <p translate>You paused your giving to these recipients. Which gifts would you like to restart? You will be able to make edits to these gifts on the next step.</p>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+
+<div class="modal-body">
+  <div class="container-fluid">
+    <div class="row row-no-spacing">
+      <div class="col-xs-12">
+        <form name="$ctrl.restartForm">
+          <div class="repeating-row select-recipient-row" ng-repeat="gift in $ctrl.gifts">
+            <gift-list-item gift="gift"
+                            selectable="checkbox">
+              <label>{{'Restart' | translate}}
+                <input type="checkbox" disabled>
+              </label>
+              <p translate>This gift cannot be changed online. For assistance,
```

Comment:
So does this component/modal step do anything other than show the user what they can't do online?